helm-charts/sawmills-collector/tests/image_digest_test.yaml

```
@@ -0,0 +1,73 @@
+templates:
+  - deployment.yaml
+  - load-balancer.yaml
+  - keda-scaler-deployment.yaml
+tests:
+  - it: should render digest instead of tag for deployment collectors
+    template: deployment.yaml
+    values:
+      - ../values.yaml
+    set:
+      image.tag: 9.9.9
+      image.digest: "sha256:deadbeef"
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)